src/main/java/com/stuypulse/robot/util/gear/EncoderIntegrator.java

```diff
@@ -0,0 +1,29 @@
+package com.stuypulse.robot.util.gear;
+
+public class EncoderIntegrator {
```

Comment:
I think it would be a lot cooler and more reliable if the `EncoderIntegrator` took in a `Encoder` class and acted as a wrapper around it. It feels a bit silly for the Drivetrain class to be managing both while this class is so small.